config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -129,6 +129,46 @@ presubmits:
           limits:
             cpu: "1"
             memory: "2Gi"
+  - name: pull-test-infra-prow-checkconfig-deprecated-job-configs
+    cluster: eks-prow-build-cluster
+    optional: true
+    decorate: true
+    run_if_changed: '^(config/prow/(config|plugins).yaml$|config/jobs/.*.yaml$)'
+    spec:
+      containers:
+      - image: gcr.io/k8s-prow/checkconfig:v20230919-a4926a4d02
+        command:
+        - sh
+        args:
+        - -c
+        - |
+          set -o errexit
+          set -o nounset
+          set -o pipefail
+          set -o xtrace
+
+          HEAD_CHECK=$(checkconfig \
+            --config-path=config/prow/config.yaml \
+            --job-config-path=config/jobs \
+            --plugin-config=config/prow/plugins.yaml \
+            --warnings=non-decorated-jobs
+
+          git checkout master
+          MASTER_CHECK=$(checkconfig \
+            --config-path=config/prow/config.yaml \
+            --job-config-path=config/jobs \
+            --plugin-config=config/prow/plugins.yaml \
+            --warnings=non-decorated-jobs
+           
+          apk add bash
+          bash -c "diff <($MASTER_CHECK) <($HEAD_CHECK)"
```

Comment:
because of this line. can't do this with sh

Reply:
```sh
$ echo "hola" > /tmp/test.1
$ echo "mund" > /tmp/test.2
$ sh -c "diff /tmp/test.1 /tmp/test.2"
1c1
< hola
---
> mund
```

Reply:
I applied your suggestions.